.github/workflows/benchmarks.yml

```diff
@@ -34,9 +34,10 @@ jobs:
         with:
           python-version: "3.10"
 
-      - uses: 'google-github-actions/auth@v2'
+      - name: Authenticate with GCP
+        uses: 'google-github-actions/auth@v2'
         with:
-          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS_COLLECTOR_SVC_ACCT }}'
+          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS_COLLECTOR_CI_VM_SVC_ACCT }}'
```

Comment:
Not something this this PR, but in a follow up it might make sense to rename this secret now that it's being used more widely than just VM creation. Also, is this the last use of the original `GOOGLE_CREDENTIALS_COLLECTOR_SVC_ACCT` secret?

Reply:
At least on this branch there are few more examples of this secret being used. Not sure how would it be after turning off eBPF though.